packages/blocks/src/api/registration.js

```diff
@@ -373,3 +373,22 @@ export const registerBlockStyle = ( blockName, styleVariation ) => {
 		};
 	} );
 };
+
+/**
+ * Unregisters a block style variation for the given block.
+ *
+ * @param {string} blockName          Name of block (example: “core/latest-posts”).
+ * @param {string} styleVariationName Name of class applied to the block.
+ */
+export const unregisterBlockStyle = ( blockName, styleVariationName ) => {
```

Comment:
Nitpick / Aside: This file should have consistent function declarations. There's no difference, at least in how we've implemented, between the function keyword and the arrow function assignment. It's not particularly relevant for what you're proposing here, but I'm raising because if we were to normalize for consistency, I would personally opt for the function keyword.

Reply:
@aduth Should eslint catch this ([if possible](https://eslint.org/docs/rules/func-style))? I can create a new issue/PR to update this since files like factory.js have also mixed declarations.

Reply:
Perhaps. It's one I've been okay to leave intentionally open-ended as a developer decision, though I can see how this might conflict with my previous comment 😄 The main worry to me was a blatant and needless inconsistency within one file, which again matters only if we consider "single voice" an ideal to strive for. 

Worth pointing out that in some cases, the use of an arrow function has meaningful consequences (on `this` value, availability of `arguments`, etc).